storage/storage.go

```diff
@@ -102,7 +104,7 @@ type PeerStore interface {
 	// filling the Snatches field is optional.
 	//
 	// If the Swarm does not exist, an empty Scrape and no error is returned.
-	ScrapeSwarm(infoHash bittorrent.InfoHash, addressFamily bittorrent.AddressFamily) bittorrent.Scrape
+	ScrapeSwarm(infoHash bittorrent.InfoHash, ip netaddr.IP) bittorrent.Scrape
```

Comment:
I wonder if this might impact performance because `netaddr.IP` is 3 words wide on a 64bit system (I think?) whereas `AddressFamily` should only be one word. I guess we should do some benchmarking at some point...

Reply:
I don't think we'll see any appreciable difference because it's all on the stack.